orgbook-service/Startup.cs

```diff
@@ -0,0 +1,199 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using System.Reflection;
+using Serilog;
+using Serilog.Exceptions;
+using System.Text;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using System.Net.Mime;
+using Newtonsoft.Json;
+using System.Linq;
+
+namespace Gov.Lclb.Cllb.OrgbookService
+{
+    public class Startup
+    {
+        public IConfiguration Configuration { get; }
+        public ILoggerFactory _loggerFactory;
+
+        public Startup(IWebHostEnvironment environment, ILoggerFactory loggerFactory)
+        {
+            _loggerFactory = loggerFactory;
+            var builder = new ConfigurationBuilder()
+                .SetBasePath(environment.ContentRootPath)
+                .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true);
+            if (!System.Diagnostics.Debugger.IsAttached)
+                builder.AddJsonFile($"appsettings.{environment.EnvironmentName}.json", optional: true);
+            builder.AddEnvironmentVariables();
+
+            if (environment.IsDevelopment())
+            {
+                builder.AddUserSecrets<Startup>();
+            }
+
+            Configuration = builder.Build();
+        }
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddIdentity<IdentityUser, IdentityRole>()
+                .AddDefaultTokenProviders();
+            
+            if (!string.IsNullOrEmpty(Configuration["JWT_TOKEN_KEY"]))
+            {
+                services.AddAuthentication(o =>
+                {
+                    o.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+                    o.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+                }).AddJwtBearer(o =>
+                {
+                    o.SaveToken = true;
+                    o.RequireHttpsMetadata = false;
+                    o.TokenValidationParameters = new Microsoft.IdentityModel.Tokens.TokenValidationParameters()
+                    {
+                        RequireExpirationTime = false,
+                        ValidIssuer = Configuration["JWT_VALID_ISSUER"],
+                        ValidAudience = Configuration["JWT_VALID_AUDIENCE"],
+                        IssuerSigningKey = new SymmetricSecurityKey(Encoding.UTF8.GetBytes(Configuration["JWT_TOKEN_KEY"]))
+                    };
+                });
+            }
+            
+            services.AddAuthorization();
+            services.AddGrpc();
+
+            services.AddHangfire(config => 
+            {
+                config.UseMemoryStorage();
+                config.UseConsole();
+            });
+
```

Comment:
A future iteration of this could use the worker service pattern rather than Hangfire.  I think it's too much to introduce that with this particular change though, as gRPC is quite a jump.

Reply:
sounds cool, looking forward to it